testsuite/tests/lexical_envs/add_to_env_mult_dests/test.py

```diff
@@ -0,0 +1,71 @@
+"""
+Test that uses an add_to_env actions with multiple env_assoc having different
+destination environments.
```

Comment:
I think this needs rewording ;-)

Reply:
Why so? (except that `actions` should be `action`)

Reply:
Ah sorry, I did read “Test” as a verb rather than a name, so I had trouble parsing this sentense. IIRC we generally document tests like functions (i.e. with imperative tense). Would you be okay with rewording this way?